code/index.html

```diff
@@ -5,32 +5,36 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" />
-    <title>Technigo Project Cluedo</title>
+    <title>Johannas Project Cluedo</title>
   </head>
   <body>
+    <h1>Welcome to the game Cluedo</h1>
+    <h3 class="text">Play by clicking on the cards.</h3>
     <section class="cards">
       <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
+        <p><strong>KILLER</strong></p>
```

Comment:
Another option for using strong three times for killer, weapon and room could have been to style them together in css with a common class.

Reply:
Agree!